test/boulder-config.json

```diff
@@ -31,21 +31,58 @@
   },
 
   "wfe": {
-    "listenAddress": "0.0.0.0:4000"
+    "listenAddress": "127.0.0.1:4000"
```

Comment:
I figure it's the right thing to do by default anyhow.

Fun story: At a previous job I found one of the frontend setup scripts opened SSH tunnels to prod, "for more realistic development." Bad enough idea as it is, right? But it turns out the tunnels were bound to 0.0.0.0. :-O